### PR TITLE
[deckhouse] add release cooldown

### PR DESCRIPTION
**Pull request by yalosev**
Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>

## Description
Add cooldown feature for a release. Via this setting we can specify, when a release will be ready to be applied in a cluster. Release will download as soon as available and will wait for the specified cooldown time

## Why do we need it, and what problem does it solve?
Sometimes we want to have more flexible release process. Ex: create release at Friday and make it available at Monday. Also we can notify users about release automatically.

## Checklist
- [x] The code is covered by unit tests.
- [ ] e2e tests are passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse
type: feature
summary: Add ability to postpone a release for applying inside a cluster
impact_level: default
```

### CODE REVIEW

File: .github/workflow_templates/deploy-channel.multi.yml [hunk 1]
```diff
@@ -35,6 +35,9 @@ on:
       editions:
         description: 'Comma separated editions to deploy. Example: ee,fe,ce'
         required: false
+      cooldown:
+        description: 'Postpone release until specified datetime (RFC3339). Example: 2026-06-06T16:16:16Z'
```
Comment (z9r5) [inline .github/workflow_templates/deploy-channel.multi.yml:39]: I think it looks inconvenient to use (T and Z must be specified).

Maybe simplify the format? E.g. YYYY-MM-DD HH:MM ?

**Review by shvgn — APPROVED** [review]
LGTM

File: .github/workflow_templates/deploy-channel.multi.yml (hunk 1 above)
Comment (yalosev) [reply .github/workflow_templates/deploy-channel.multi.yml]: Done. We can use any of these formats:
- `YYYY-MM-DD HH:MM`
- `YYYY-MM-DD HH:MM:SS`
- `RFC3339`

everything is supported

Comment (yalosev) [reply .github/workflow_templates/deploy-channel.multi.yml]: `YYYY-MM-DD HH:MM` is set as default and put into docs

**Review by konstantin-axenov — APPROVED** [review]

**Pull request merged**